mobile/src/bundles/common/components/checkbox-in-group/checkbox-in-group.tsx

```diff
@@ -0,0 +1,50 @@
+import CheckBox, { type CheckBoxProps } from '@react-native-community/checkbox';
+import React from 'react';
+import { type StyleProp, type ViewStyle } from 'react-native';
+
+import { Text, View } from '~/bundles/common/components/components';
+import { Color, TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+type Properties = CheckBoxProps & {
```

Comment:
I guess we can modify default checkbox, because we now have 2 similar components

Reply:
done